client/tailwind.config.js

```diff
@@ -68,6 +70,15 @@ module.exports = {
       none: 'none',
     },
     spacing,
+    extend: {
+      width: {
+        sidebar: '200px',
```

Comment:
I was trying to make a push further down the tailwind well here haha. I guess we aren't ready for this yet but one day when we want to go full tailwind this would be a good option for this value.